examples/workflows/README.md

```diff
@@ -0,0 +1,45 @@
+## Example workflows plugin
+
+This example plugin demonstrates how to create a workflows plugin to automatically approve/deny new access requests based on a simple Allow List.
+
+## Demo
+
+This demo can be used to quickly get the example plugin up and running.
+
+##### Create resources
+
+```bash
```

Comment:
I'm not sure `bash` is the correct highlighting mode for this and below "shell sessions" as these aren't actually bash scripts. Apparently `console` should be a better choice.